README.md

```diff
@@ -0,0 +1,83 @@
+# PyVista Tutorial
+
+## Abstract
+
+This tutorial will introduce 3D data visualization using PyVista (https://github.com/pyvista/pyvista).
+PyVista offers 3D plotting and mesh analysis through a streamlined interface for the Visualization Toolkit (VTK).
+PyVista uses VTK (http://www.vtk.org) under the covers and provides a very Python-friendly API.
+
+At the end of the tutorial users will be able to
+
+- Use PyVista effectively to create 3D visualizations. 
+- Understand the basic class structure of PyVista.
+
+## Keywords
+
+[visualization](https://github.com/topics/visualization)
+[meshviewer](https://github.com/topics/meshviewer)
+[vtk](https://github.com/topics/vtk)
+[3d](https://github.com/topics/3d)
+
+## Other Information and Files
+
+### Tutorial Description.
+Same as Abstract.
+
+### Tutorial Outline.
+
+0. Getting Started - Why PyVista? (10 min for talk, 10 min for exercise)
+
+1. Basic usage - Reading and plotting 3D data.  (10 min for talk, 10 min for exercise)
+
+2. What is a Mesh? - Learn about "Point", "Cell" and "Attributes". (10 min for talk, 10 min for exercise)
+
+3. PyVista Data Model - Learn about the data model of the PyVista framework. (10 min for talk, 5 min for exercise)
+
+4. Mesh Creation - Exercise to build and plot meshes in PyVista. (10 min for talk, 5 min for exercise)
+
+6. Mesh animation - Show how to animate meshes using the PyVista API. (10 min for talk, 5 min for exercise)
+
+7. Unique uses of PyVista -  About some unique applications of PyVista. (10 min for talk, 5 min for exercise)
+
+8. External Examples - How PyVista is used in external projects. (15 min for talk)
```

Comment:
Talk should be exactly 4 hours. Please verify the times.

```suggestion
0. Getting Started - PyVista for 3D Visualization within Python. (10 min for talk, 10 min for exercise)

1. Basic usage - Reading and plotting 3D data using built-in mesh data and external files.  (10 min for talk, 10 min for exercise)

2. What is a Mesh? - Learn about the PyVista data types and how you can use it to visualize almost anything. (15 min for talk, 10 min for exercise)

3. PyVista Data Model - Learn about the data model of the PyVista framework and how you can visualize your data in style using point, cell, and field data. (15 min for talk, 10 min for exercise)

4. Break. Stretch fingers and grab some coffee. (15 minutes)

5. Mesh animation - Show how to animate meshes using the PyVista API to make awesome online documentation. (10 min for talk, 10 min for exercise)

6. PyVista, jupyterlab, and sphinx - Leverage PyVista to make some awesome interactive web documentation. (20 min for talk, 10 min for exercise)

7. Quick break. Prepare for the final run. (10 minutes)

8. PyVista & VTK - Show how PyVista uses VTK and how you can combine the best of both worlds! (10 minutes for talk, 10 minutes for exercise)

9. PyVista and Qt - Demonstrate how to use PyVista to create standalone applications using pyinstaller and the Qt framework. (15 min for talk, 10 for exercise)

10. PyVista in Action - Show how PyVista is already being used within several projects and can be used for all things visualization. (15 min for talk)

11. Open up to Questions. (15 minutes)
```

Reply:
Thanks a lot. I could feel your rich tutorial experience from the modified content.